procrastinate/cli.py

```diff
@@ -77,14 +69,7 @@ def print_version(ctx, __, value):
     count=True,
     help="Use multiple times to increase verbosity",
 )
-@click.option(
-    "-V",
-    "--version",
-    is_flag=True,
-    callback=print_version,
-    expose_value=False,
-    is_eager=True,
-)
+@click.version_option(None, '-V', '--version')
```

Comment:
So this:
```suggestion
@click.version_option('-V', '--version', version=None)
```
doesn't work ?

Reply:
Nope:
`version_option() got multiple values for argument 'version'`

Reply:
Ah yes. ok.